README.md

```diff
@@ -15,3 +15,6 @@
 
 [coveralls-img]: https://coveralls.io/repos/github/JuliaStats/GLM.jl/badge.svg?branch=master
 [coveralls-url]: https://coveralls.io/github/JuliaStats/GLM.jl?branch=master
+
+[DOI-img]: https://zenodo.org/badge/DOI/10.5281/zenodo.3376014.svg
```

Comment:
Better use the DOI referring to all version of GLM: we don't want to update the README for every new release. Users can find the DOI for the version they are using on Zenodo.

```suggestion
[DOI-img]: https://zenodo.org/badge/DOI/10.5281/zenodo.3376013.svg
```